lib/console.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
 
 const util = require('util');
+const errors = require('internal/errors');
 
 function Console(stdout, stderr) {
   if (!(this instanceof Console)) {
     return new Console(stdout, stderr);
   }
   if (!stdout || typeof stdout.write !== 'function') {
-    throw new TypeError('Console expects a writable stream instance');
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'stdout', 'WriteStream');
```

Comment:
Hmm, wait, `WriteStream` is not an actual type or class name (at leat the constructor of writable stream from our `stream` module is named `Writable`). Is it OK to pass a descriptive type (like "writable stream") to the formatter of `ERR_INVALID_ARG_TYPE`? @jasnell